.github/workflows/auto_deployment_on_label.yml

```
@@ -0,0 +1,71 @@
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Start deployment
+        if: steps.check_build.outputs.skip != 'true'
+        uses: bobheadxi/deployments@v1.5.0
+        id: gh_deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: ${{ github.head_ref }}
+      - name: Trigger autodeployment hook
+        if: steps.check_build.outputs.skip != 'true'
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)